### PR TITLE
New hook function onSubmitted

### PR DESCRIPTION
**Pull request by commelius-grela**
New hook function `onSubmitted` - complementary to the `onCannotSubmit`
At the moment extending class needs to override `onSubmitClicked` when easier is to have a hook function to be overridden

### CODE REVIEW

File: src/core/js/views/questionView.js [hunk 1]
```diff
@@ -194,6 +195,8 @@ define([
         // Blank method for question to fill out when the question cannot be submitted
         onCannotSubmit: function() {},
 
+        // Blank method for question to fill out when the question was succesfully submitted
```
Comment (tomgreenfield) [inline src/core/js/views/questionView.js]: Small typo on ‘successfully’

**Comment by tomgreenfield** [comment]
+1

**Comment by moloko** [comment]
+1

**Comment by brian-learningpool** [comment]
+1

**Pull request merged**